CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+<a name="6.3.0"></a>
+# [6.3.0](https://github.com/reactstrap/reactstrap/compare/6.1.0...6.3.0) (2018-07-10)
```

Comment:
just noticed this when creating the release tag. The link has `6.1.0...6.3.0` but it should be `6.2.0...6.3.0`. Maybe you didn't have all of the tags when running the command?

Reply:
Yes, I think that is was happened.
In my clone (from my fork), I have a `base` pointing to `git@github.com:reactstrap/reactstrap.git`
And I did just `git pull base master` when I should have done `git pull base master --tags`

I had to edit the CHANGELOG completely (It was including the same changes than v6.2.0 plus the latest nes) to include these 2 commits, and I think this is why
Maybe a note (in the Releasing section) could be added to README pointing this